python/kserve/kserve/model_server.py

```diff
@@ -360,12 +341,17 @@ def default_exception_handler(
         """Default exception handler for event loop.
 
         This is called when an exception occurs and no exception handler is set.
-        By default, this will shut down the server gracefully.
-
         This can be called by a custom exception handler that wants to defer to the default handler behavior.
         """
-        # gracefully shutdown the server
-        loop.run_until_complete(self.stop())
```

Comment:
Do we still need to shut down the server?

Reply:
No Need. It will be handled by uvicorn.